azurepipelines/e2e_test/scenarios/ubuntu-20.04-arm64/vm_setup/devicesetup.py

```diff
@@ -0,0 +1,83 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import json
+import sys
+
+# Note: the intention is that this script is called like:
+# python ./<scenario-name>/vm-setup/devicesetup.py
+sys.path.append('./scenarios/')
+from testingtoolkit import DeviceUpdateTestHelper,DuAutomatedTestConfigurationManager
+
+# Note: the intention is that this script is called like:
+# python ./scenarios/<scenario-name>/<test-script-name>.py
+sys.path.append('./scenarios/ubuntu-20.04-arm64/')
```

Comment:
as above.

Reply:
It looks ugly and it is. Python projects like to be flat but it's needed for the way we're using it here. Python runs as a command-line tool so it bows to those values.


This is a good candidate for code quality improvements. Tracked here: https://microsoft.visualstudio.com/OS/_queries/edit/43984595/?triage=true&workitem=44136731